docs/run/primitives.mdx

```diff
@@ -80,8 +80,8 @@ estimator.run([circuit1, circuit2, ...],[observable1, observable2, ...],[param_v
 
 #### Estimator V2
 
-* Takes one parameter: PUBs in the format (`<circuit>`, `<observables>`, `<parameter values>`, `<precision>`)
-* Numpy [broadcasting rules](https://numpy.org/doc/stable/user/basics.broadcasting.html) are used when combining observables and parameter values.
+* The `run()` method takes an array of PUBs. Each PUB is in the format (`<single circuit>`, `<one or more observables>`, `<optional one or more parameter values>`, `<optional precision>`)
```

Comment:
I think that the clarification made for the sampler section about num.circuits/parameters also fits well here (repetition doesn't hurt). It's also good to note that each item of "parameter_values" might contain one or more parameters. What I mean here is that for a circuit with 2 parameters, a pub with 1 circuit, 1 observable and 1 "parameter value" would look like this:

`estimator.run([(circuit, observable, [0.5, 0.5])])`

While 2 "parameter values" would look like this:

`estimator.run([(circuit, observable, [[0.5, 0.5], [0.6, 0.7]])])`

So when we commonly talk about parameter values, we are not talking about individual parameters, we are talking about "sets of parameters" or "arrays of parameters", and the number of parameters in a set depends on the number of parameters in the circuit. Anyway, this is all to preface this small suggestion:

```suggestion
* The `run()` method takes an array of PUBs. Each PUB is in the format (`<single circuit>`, `<one or more observables>`, `<optional one or more parameter values>`, `<optional precision>`). Note that there is only one circuit per PUB, there can be multiple items in `observables` and `parameter values`, and each item of `parameter values` can be a list or a single parameter.
```